docs/config_entries_config_flow_handler.md

```diff
@@ -70,6 +70,17 @@ await self.async_set_unique_id(device_unique_id)
 self._abort_if_unique_id_configured()
 ```
 
+Should the config flow then abort, the text resource with the key `already_configured` from the `abort` part of your `strings.json` will be displayed to the user in the interface as an abort reason.
+
+```json
+{
+  "config": {
+    "abort": {
+      "already_configured": "The device is already configured."
```

Comment:
That's our standard text from our common reference strings.
```suggestion
      "already_configured": "Device is already configured"
```

Reply:
thanks